### DISCUSSION TITLE
Repeated Images don't work<title>

### DISCUSSION THREAD

**Discussion by KarunamoorthySasvin3** [community/community#58378] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

When I reused the same image, it appears once and doesn't appear the second time. When I inspect the page I see this, does anyone know how to fix this?

(example of the inspect) 
<img src="images/early_war_heading.png" alt="Image of 2 guns" data-nsfw-filter-status="sfw" style="visibility: visible;">

<img src="images/early_war_heading.png" alt="Image of 2 guns" data-nsfw-filter-status="processing" style="visibility: hidden;">

**Comment by DSDmark** [comment]
Hi @KarunamoorthySasvin3,

Can you **specific more** about the details of what you want? For better understanding 😅. Also, you can add some **screenshots** or **error** which you were facing 👍.

**Reply by KarunamoorthySasvin3** [reply]
I ended up figuring it out but thanks, it was a problem with my browser.

**Reply by DSDmark** [reply]
Hi @KarunamoorthySasvin3,

😊 I hope you get your answer. If you get your response, you should close that question by clicking ✅ accept answer button, or you can ✍️ write the answer and accept your own answer.

> **Note**: if you close this question, then nobody will contribute to your question anymore. So, close this question when you satisfy your answer, or you can write your own question with a better explanation

**Comment by KarunamoorthySasvin3** [comment, marked as answer]
The problem is a broken nsfw filter due to me using a school account, the image is unable to be processed through the filter if it is used more then once.
